src/node_buffer.cc

```diff
@@ -541,6 +659,121 @@ void WriteDoubleBE(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+template <typename T, enum Endianness endianness,
+         T (*strtoT)(const char*, char**, int)>
+uint32_t WriteInt64Generic(const FunctionCallbackInfo<Value>& args) {
```

Comment:
Sorry, but I don't want to deal with this in my rewrite. So I'm going to ask that you do some forward-proofing. Basically, in the future Buffers will allow arbitrary memory amounts to be allocated, so a `uint32_t` won't be big enough to handle all cases.